packages/react-components/react-infobutton/src/components/InfoLabel/useInfoLabel.ts

```diff
@@ -48,13 +48,24 @@ export const useInfoLabel_unstable = (props: InfoLabelProps, ref: React.Ref<HTML
     required: !!info,
     defaultProps: {
       id: useId('infobutton-'),
-      info,
       size,
+      info,
+    },
+  });
+
+  const infoButtonInfo = resolveShorthand(infoButton?.info, {
+    defaultProps: {
+      id: useId('infobutton-info-'),
```

Comment:
This is optional to do, but in the case of a single element having several generated IDs, I prefer to generate a base ID and then append the slot name to it. This both helps debug (easier to see where the ID came from in dev tools), and reduces the number of calls to `useId`. (See `useField` for an example)

Like:
```ts
const baseId = useId('infolabel-');

// For the label slot:
id: baseId + '__label',

// For the infoButton slot:
id: baseId + '__infoButton',

// For the info slot:
id: baseId + '__info',
```